cocos/render-scene/scene/camera.ts

```diff
@@ -996,6 +996,12 @@ export class Camera {
         if (!this._node) return;
 
         let viewProjDirty = false;
+        const xr = globalThis.__globalXR;
+        if (xr.isWebXR && xr.webXRWindowMap && xr.webXRMatProjs) {
```

Comment:
Where is `webXRMatProjs`  assgined?

Reply:
When users use the webxr of xr plugin, values will be assigned at runtime.